test/testcafe/spec/ChallengeOktaVerifyPush_spec.js

```diff
@@ -25,6 +29,16 @@ const pushWaitMock = RequestMock()
   .onRequestTo('http://localhost:3000/idp/idx/challenge/poll')
   .respond(pushPoll);
 
+const sendPushMock = RequestMock()
```

Comment:
nit: change the names of this and the next constant to reflect they have autoChallenge. Maybe `pushAutoChallengeMock` and `pushWaitAutoChallengeMock`?